client/a8c-for-agencies/sections/purchases/invoices/invoices-list-card/style.scss

```diff
@@ -0,0 +1,20 @@
+@import "@wordpress/base-styles/breakpoints";
+@import "@wordpress/base-styles/mixins";
+
+.invoices-list-card__actions {
+	text-align: left;
+
+	@include break-xlarge() {
+		text-align: right;
+	}
+
+	.button {
+		&:focus {
+			border-width: 1px;
+		}
+
+		+ .button {
+			margin-left: 12px;
```

Comment:
To ensure that the CSS property is RTL-friendly, we can use `margin-inline-start` here.